src/main/java/org/terracotta/tinypounder/TinyPounderMainUI.java

```diff
@@ -1192,25 +1194,25 @@ private void generatePropertiesFile(boolean skipConfirmOverwrite) {
           sb.append(nodePrefix + "hostname=" + hostName.getValue() + "\n");
           sb.append(nodePrefix + "port=" + clientPort.getValue() + "\n");
           sb.append(nodePrefix + "group-port=" + groupPort.getValue() + "\n");
-          sb.append(nodePrefix + "log-dir=" + logs.getValue() + "\n");
+          sb.append(nodePrefix + "log-dir=" + logs.getValue().replace("\\", "\\\\") + System.lineSeparator());
```

Comment:
Do you think it would be better to translate those paths to forward-slash (Unix)? I believe that works and would be platform-independent. Also, maybe create a helper static method `private static String getPath(Field field)` so this translating is happening in one place.

Reply:
I'll make the change.  Windows does recognize a forward slash.